snap/snapcraft.yaml

```diff
@@ -552,6 +656,7 @@ parts:
   support-rulesengine:
     source: https://github.com/edgexfoundry/support-rulesengine.git
     source-branch: "master"
```

Comment:
Shouldn't this be updated to "edinburgh"?

Reply:
Changed